index.ts

```diff
@@ -112,15 +112,6 @@ try {
 
     await exec(changesetBinary, ['version', '--snapshot', versionPrefix]);
 
-    // Run after `changeset version` so build scripts can use updated versions
-    if (buildScript) {
-      const commands = buildScript.split('&&').map((cmd) => cmd.trim());
-      for (const cmd of commands) {
-        const [cmdName, ...cmdArgs] = cmd.split(/\s+/);
-        await exec(cmdName, cmdArgs);
-      }
-    }
-
     const {packages} = await getPackages(process.cwd());
     const snapshots = [];
     packages.forEach(({packageJson}) => {
```

Comment:
`package.json` files might not exist after the `build_script` is ran. So it's important to get this information first.